sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs

```diff
@@ -610,14 +610,29 @@ public async Task AcquireLeaseAsync()
             });
         }
 
+        [Test]
+        public async Task AcquireLeaseAsync_ErrorDurationTooLarge()
+        {
+            // Arrange
+            var service = this.GetServiceClient_SharedKey();
+            var container = this.InstrumentClient(service.GetBlobContainerClient(this.GetNewContainerName()));
+            var id = this.Recording.Random.NewGuid().ToString();
+            var duration = TimeSpan.MaxValue;
+
+            // Assert
+            await TestHelper.AssertExpectedExceptionAsync<StorageRequestFailedException>(
+                container.GetLeaseClient(id).AcquireAsync(duration: duration),
+                e => Assert.AreEqual("InvalidHeaderValue", e.ErrorCode.Split('\n')[0]));
```

Comment:
Nit - Probably safer to do a `StringAssert.Contains` so future formatting changes don't break the test _(I know a lot of nearby tests probably do this too though)_

Reply:
Good idea.